justfile

```diff
@@ -192,6 +218,22 @@ mind-benchmark kind:
     cargo test --package xayn-web-api --release --lib \
         -- --nocapture --include-ignored --exact mind::run_{{kind}}_benchmark
 
+_test-project-root:
+    echo -n {{justfile_directory()}}
+
+_test-generate-id:
+    echo -n "t$(date +%y%m%d_%H%M%S)_$(printf "%04x" "$RANDOM")"
+
+_test-create-dbs $TEST_ID:
+    #!/usr/bin/env -S bash -eux -o pipefail
+    psql -c "CREATE DATABASE ${TEST_ID};" postgresql://user:pw@localhost/xayn 1>&2
+    ./web-api/elastic-search/create_es_index.sh "http://localhost:9200/${TEST_ID}"
```

Comment:
After this you can assess the dbs as:

- pg: `postgresql://user:pw@localhost/${TEST_ID}`
- es: `http://localhost:9200/${TEST_ID}`

After a successful test it's best to call `just _test-drop-dbs $TEST_ID` to drop the db/index.